go/libraries/doltcore/schema/typeinfo/point.go

```diff
@@ -208,3 +209,25 @@ func pointTypeConverter(ctx context.Context, src *pointType, destTi TypeInfo) (t
 		return nil, false, UnhandledTypeConversion.New(src.String(), destTi.String())
 	}
 }
+
+func CreatePointTypeFromParams(params map[string]string) (TypeInfo, error) {
+	var (
+		err     error
+		sridVal uint64
+		def     bool
+	)
+	if s, ok := params["SRID"]; ok {
```

Comment:
adding check here because the previously created tables with geometry type will not have value in it, so starting server fails.

Reply:
smart; good job thinking about backwards compatibility with existing databases.